spa/src/components/donations/filters/StatusFilter.js

```diff
@@ -7,22 +7,25 @@ import { PAYMENT_STATUS } from 'constants';
 
 const STATUS_FILTERS = Object.values(PAYMENT_STATUS);
 
-function StatusFilter({ filter = [], handleFilterChange }) {
+function StatusFilter({ filter = [], handleFilterChange, excludeStatusFilters = [] }) {
   return (
     <FilterWrapper data-testid="status-filter">
       <S.StatusFilter>
         <FilterLabel>Status:</FilterLabel>
         <S.Statuses>
-          {STATUS_FILTERS.map((f) => (
-            <S.StatusBadge
-              key={f}
-              data-testid={`status-filter-${f}`}
-              selected={filter.includes(f)}
-              onClick={() => handleFilterChange('status', f)}
-            >
-              <StatusCellIcon status={f} showText size="sm" />
-            </S.StatusBadge>
-          ))}
+          {STATUS_FILTERS.map(
+            (f) =>
+              excludeStatusFilters.indexOf(f) == -1 && (
+                <S.StatusBadge
```

Comment:
I would prefer to use something like the `filter` method:
You can add the `!` to choose if you want to select the subset, or if you want to exclude the subset values.

![image](https://user-images.githubusercontent.com/9381325/186000505-6608f058-1266-4afe-bb73-b4e88f7019d0.png)
```suggestion
          {STATUS_FILTERS.filter(status => excludeStatusFilters.includes(status)).map(
            (status) => (
                <S.StatusBadge
```